cmd/substate-cli/main.go

```diff
@@ -34,6 +34,9 @@ func init() {
 	app.Flags = []cli.Flag{}
 	app.Commands = []cli.Command{
 		replay.ReplayCommand,
+		replay.ReplayStorageCommand,
```

Comment:
Perhaps we rename ReplayStorageCommand => GetStorageUpdateSizeCommand

Reply:
Done